### PR TITLE
Nextra 2 Blogpost

### PR DESCRIPTION
**Pull request by B2o5T**

### CODE REVIEW

File: website/pages/blog/announcing-nextra-2.mdx [hunk 1]
```diff
@@ -0,0 +1,344 @@
...
+<NextImage
+  src={nextraContrastMode}
+  className="mt-6 rounded-3xl border border-gray-700"
+  alt="Example of Nextra page with contrast mode enabled"
+  loading="lazy"
+/>
+
+## Conclusion
+
+[The Guild](/) discovered Nextra 2 from the first betas and actively participated in development and improvement this library.
```
Comment (Urigo) [inline website/pages/blog/announcing-nextra-2.mdx:340]: maybe also mention that you are now part of the Nextra team and link here?
https://nextra.site/about#team

File: website/pages/blog/announcing-nextra-2.mdx [hunk 2]
```diff
@@ -0,0 +1,344 @@
...
+At the end we migrated all of our documentation websites, and received a bunch of positive feedback
+from our community ❤️.
```
Comment (Urigo) [inline website/pages/blog/announcing-nextra-2.mdx:342]: maybe you can also expand more on The Guild's docs and all the difference features we've added (like Giscus for example)

**Comment by Urigo** [comment]
@B2o5T maybe you can also ask @shuding to review?
or maybe they also want to publish a blog post?

**Review by dotansimha — APPROVED** [review]

**Review by enisdenjo — APPROVED** [review]

**Review by n1ru4l — APPROVED** [review]

**Comment by saihaj** [comment]
CleanShot 2023-01-05 at 10 28 38

**Pull request merged**